v21.1/take-and-restore-locality-aware-backups.md

```diff
@@ -25,31 +25,55 @@ A locality-aware backup is specified by a list of URIs, each of which has a `COC
 The locality query string parameters must be [URL-encoded](https://en.wikipedia.org/wiki/Percent-encoding).
 {{site.data.alerts.end}}
 
-During locality-aware backups, backup file placement is determined by leaseholder placement, as each node is responsible for backing up the ranges for which it is the leaseholder.  Nodes write files to the backup storage location whose locality matches their own node localities, with a preference for more specific values in the locality hierarchy.  If there is no match, the `default` locality is used.
+During locality-aware backups, backup file placement is determined by [leaseholder](architecture/replication-layer.html#leases) placement, as each node is responsible for backing up the ranges for which it is the leaseholder. Nodes write files to the backup storage location whose locality matches their own node localities, with a preference for more specific values in the locality hierarchy. If there is no match, the `default` locality is used.
```

Comment:
I found this a bit hard to read, I wonder if it would make sense to unpack this into two parts.

- Since each node is responsible for backing up the ranges for which it is the leaseholder, in a locality-aware backup,  the locality of the leaseholder node determines where the backup files will be placed.

- The leaseholder node writes files to the backup storage location whose locality matches it's own localities, which a preference for more specific values in the locality hierarchy.